bigtable-hbase/pom.xml

```diff
@@ -41,19 +41,12 @@ limitations under the License.
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId>
             <version>${netty.version}</version>
+            <scope>test</scope>
```

Comment:
Maven compiled fine without this, probably because of the grpc depdencies.  I'll make those dependencies explicit.